sci-libs/minc-toolkit/minc-toolkit-9999.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="5"
+
+inherit cmake-utils git-r3
+
+DESCRIPTION="Metaproject uniting all the available MINC tools"
+HOMEPAGE="https://github.com/BIC-MNI/minc-toolkit"
+SRC_URI=""
+EGIT_REPO_URI="git://github.com/BIC-MNI/minc-toolkit.git"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE="itk"
+KEYWORDS=""
+
+COMMON_DEP="dev-libs/libpcre
+	sci-libs/fftw
+	sci-libs/gsl
+	sci-libs/hdf5
+	itk? ( sci-libs/itk )
```

Comment:
No USE dependent dependencies if there are no buildtime changes. Use optfeatures from eutils.eclass for this,